psi-reports/report-2021-02-12.json

```diff
@@ -0,0 +1,65 @@
+{
+  "timestamp": "2021-02-12T07:35:28.027Z",
+  "reports": [
+    {
+      "url": "https://jefrydco.id/",
+      "device": "mobile",
+      "perf": 0.91,
+      "fid": 0,
+      "lcp": 909,
+      "cls": 0.007668701171875,
+      "fcp": 844,
+      "fci": 3664,
+      "tbt": 436,
+      "tti": 3709,
+      "si": 1599.7016304882254,
+      "req": 18,
+      "size": 266373
+    },
+    {
+      "url": "https://jefrydco.id/",
+      "device": "desktop",
```

Comment:
It seems failed to get  desktop score.

Reply:
Is there any white space on the device `desktop`.
I think i was forget to trim the value

Reply:
Do you have set secret_key for PSI_API_KEY?

Reply:
All the score getting zero value 😭